geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractIncrByFloatIntegrationTest.java

```diff
@@ -15,24 +15,25 @@
 package org.apache.geode.redis.internal.executor.string;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-import java.util.Random;
+import java.math.BigDecimal;
 
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 import redis.clients.jedis.Jedis;
+import redis.clients.jedis.Protocol;
 
 import org.apache.geode.test.dunit.rules.RedisPortSupplier;
 
 public abstract class AbstractIncrByFloatIntegrationTest implements RedisPortSupplier {
```

Comment:
It would be nice to add a test making sure the correct error is returned when we try to do incrby on a key whose value is not a string.

Reply:
I think I have that now. Let me know if you think it should cover other data types too.